src/main/java/apoc/map/Maps.java

```diff
@@ -178,18 +178,39 @@ public Object get(@Name("map") Map<String,Object> map, @Name("key") String key,
     }
 
     @UserFunction
-    @Description("apoc.map.removeKey(map,key)")
-    public Map<String,Object> removeKey(@Name("map") Map<String,Object> map, @Name("key") String key) {
+    @Description("apoc.map.removeKey(map,key,recursive:false) - remove the key from the map (recursively if recursive is true)")
+    public Map<String,Object> removeKey(@Name("map") Map<String,Object> map, @Name("key") String key, @Name(value="recursive", defaultValue = "false") boolean recursive) {
```

Comment:
Can you turn the `recursive` argument into a map `{recursive: true/false}`, so in the future we can add more properties like the recursion `depth`.
```suggestion
    public Map<String,Object> removeKey(@Name("map") Map<String,Object> map, @Name("key") String key, @Name(value="config", defaultValue = "{}") Map<String, Object> config) {
```

Reply:
the same for `removeKeys`